src/cr/cube/cubepart.py

```diff
@@ -268,6 +268,16 @@ def __repr__(self):
 
     # ---interface ---------------------------------------------------
 
+    @lazyproperty
```

Comment:
Can you do the type annotation for mypy while you're at it? Only for your new work I mean...

Reply:
sure